styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -216,33 +254,73 @@ $default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.
 }
 
 .storyOverview-itemInner {
-  border: 1px solid $gray-light;
-  padding: 5px;
+  border: 1px solid fade-out($white, .7);
```

Comment:
In my opinion, all colors should be variables in the `swatch.scss` file, even derivative colors like this. Otherwise we get inconsistencies across the site.  

In this PR alone we have fade-outs on white at factors of (.3, .7, .9). Then some other PR someone could decide they want it at (.2, .6, .8). That's just gonna be a pain to manage.